sbt-tasks/test/com/commercetools/sunrise/theme/WebjarsFilesCopierTest.java

```diff
@@ -15,15 +15,15 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class WebjarsFilesCopierTest {
-    private static final String DEST_PATH = WebjarsFilesCopierTest.class.getClass().getResource("/com/commercetools/sunrise/theme").getPath();
+
+    private static final String DEST_PATH = WebjarsFilesCopierTest.class.getClass().getResource("/").getPath() + "files";
```

Comment:
Not sure why I used as dest folder the one with the class. Now I have defined a different folder, only for this purpose, as destination path. This way we can clean its contents cleanly. WDYT @schleichardt ?